plugins/github-actions/src/components/WorkflowRunsCard/WorkflowRunsCard.tsx

```diff
@@ -0,0 +1,416 @@
+  createStyles({
+    card: {
+      border: `1px solid ${theme.palette.divider}`,
+      boxShadow: theme.shadows[2],
+      borderRadius: '4px',
+      overflow: 'visible',
+      position: 'relative',
+      margin: theme.spacing(4, 1, 1),
+      flex: '1',
+      minWidth: '0px',
+    },
+    externalLinkIcon: {
+      fontSize: 'inherit',
+      verticalAlign: 'middle',
+    },
+    bottomline: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      marginTop: '-5px',
+    },
+    pagination: {
+      width: '100%',
+    },
+  }),
+);
+
+type WorkflowRunsCardViewProps = {
+  runs?: WorkflowRun[];
+  searchTerm: string;
+  loading: boolean;
+  onChangePageSize: (pageSize: number) => void;
+  onChangePage: (page: number) => void;
+  page: number;
+  total: number;
+  pageSize: number;
+  projectName: string;
+};
+
+const statusColors: Record<string, string> = {
+  skipped: 'warning',
+  canceled: 'info',
+  timed_out: 'error',
+  failure: 'error',
+  success: 'success',
+};
+
+const matchesSearchTerm = (run: WorkflowRun, searchTerm: string) => {
+  const lowerCaseSearchTerm = searchTerm.toLocaleLowerCase();
+  return (
+    run.workflowName?.toLocaleLowerCase().includes(lowerCaseSearchTerm) ||
+    run.source.branchName?.toLocaleLowerCase().includes(lowerCaseSearchTerm) ||
+    run.status?.toLocaleLowerCase().includes(lowerCaseSearchTerm) ||
+    run.id?.toLocaleLowerCase().includes(lowerCaseSearchTerm)
+  );
+};
+
+export const WorkflowRunsCardView = ({
+  runs,
+  searchTerm,
+  loading,
+  onChangePageSize,
+  onChangePage,
+  page,
+  total,
+  pageSize,
+}: WorkflowRunsCardViewProps) => {
+  const classes = useStyles();
+  const routeLink = useRouteRef(buildRouteRef);
+
+  const filteredRuns = runs?.filter(run => matchesSearchTerm(run, searchTerm));
+
+  return (
+    <Grid container spacing={3}>
+      {filteredRuns && runs?.length !== 0 ? (
+        filteredRuns.map(run => (
+          <Grid key={run.id} item container xs={12} sm={4} md={4} xl={4}>
+            <Box className={classes.card}>
+              <Box
+                display="flex"
+                flexDirection="column"
+                m={3}
+                alignItems="center"
+                justifyContent="center"
+              >
+                <Box pt={2} sx={{ width: '100%' }} textAlign="center">
+                  <Tooltip
+                    title={run.status ?? 'No Status'}
+                    placement="top-start"
+                  >
+                    <Alert
+                      variant="outlined"
+                      severity={
+                        statusColors[
+                          run.conclusion as keyof typeof statusColors
+                        ] as Color
+                      }
+                      style={{ alignItems: 'center' }}
+                    >
+                      <Typography variant="h6">
+                        <Link
+                          component={RouterLink}
```

Comment:
this really shouldn't be necessary (but also shouldn't hurt); Link already does this internally, dynamically, depending on the `to`.